### ISSUE TITLE
Remove html tags from comments

### ISSUE THREAD

**Issue by tatianacv** [ponder-lab/GitHub-Issue-Classifier#49]
In some comment lines for the issues, they appear html tags (e.g. li). Please remove these.

**Comment by khatchad** [comment]
Can you provide an example in the dataset?

**Comment by tatianacv** [comment]
I will use issue 761588798 as an example. In the line comments, we have ``p><em sourced URL tensorflow tensorflow releases">tensorflow releases</a>.</em></p``, ``p><code tf.distribute</code introduces experimental support asynchronous training kera model via URL api_docs python tf distribute experimental parameterserverstrategy?version nightly"><code tf.distribute.experimental parameterserverstrategy</code></a api please see additional details.</p``. These HTML tags are unnecessary.

**Comment by y3pio** [comment]
Hi @tatianacv do you think you can link the issue URL as well, like you did for the other issue? I can't seem to find `761588798` on the sample classified result files that I attached to the asana tasks.

**Comment by tatianacv** [comment]
Of course, for that one the URL is https://github.com/animesh/deepmind-research/pull/5.

**Comment by y3pio** [comment]
@tatianacv can you confirm if that's the right URL? I can't seem to find the `HTML` code that you posted above in the issue you linked: https://github.com/animesh/deepmind-research/pull/5.

i.e Searching for strings like `tensorflow releases` or `training kera model via URL...` doesn't show any matches.

**Comment by tatianacv** [comment]
@y3pio You have to click on release notes, changelog and commits for the text to show up. The `<code tf.distribute</code introduces experimental support asynchronous training` is the first bullet point below **Release notes**.

**Comment by y3pio** [comment]
Ah I see it now, thanks! Will look into this.
